### PR TITLE
Fix test cases after #34

### LINKED ISSUE #39: 11 test cases failed  after merge #34

### PR DESCRIPTION
**Pull request by huningxin**
Fix #39 

Remove the tests for unexposed methods.

### CODE REVIEW

**Comment by huningxin** [comment]
@ganwenyao , please review. Thanks!

File: modules/js/test/test_objdetect.js [hunk 1]
```diff
@@ -40,43 +40,19 @@ if (typeof module !== 'undefined' && module.exports) {
...
-			groupThreshold = 1,
-			eps = 0.2;
-
-		let rect1 = new cv.Rect(1, 2, 3, 4),
-			rect2 = new cv.Rect(1, 4, 2, 3);
-
-		rectList.push_back(rect1);
-		rectList.push_back(rect2);
-
-		cv.groupRectangles(rectList, weights, groupThreshold, eps);
```
Comment (ganwenyao) [inline modules/js/test/test_objdetect.js]: groupRectangles should be exposed but not

File: modules/js/test/test_objdetect.js [hunk 2]
```diff
@@ -40,43 +40,19 @@ if (typeof module !== 'undefined' && module.exports) {
...
-		rect1.delete();
-		rect2.delete();
-	}
-
 	// CascadeClassifier
 	{
 		let classifier = new cv.CascadeClassifier(),
 			modelPath = '/haarcascade_frontalface_default.xml';
 
-		assert.equal(classifier.empty(), true);
```
Comment (ganwenyao) [inline modules/js/test/test_objdetect.js]: empty should be exposed but not

**Comment by ganwenyao** [comment]
2 changes are not shown in *Files changed*:
1. imwrite,imread in test_io.js should be exposed but not
2. BackgroundSubtractorMOG2 in test_video.js has error
@huningxin

File: modules/js/test/test_objdetect.js (hunk 1 above)
Comment (huningxin) [reply modules/js/test/test_objdetect.js]: Good catch! I will update the white list and keep this test.

File: modules/js/test/test_objdetect.js (hunk 2 above)
Comment (huningxin) [reply modules/js/test/test_objdetect.js]: Ditto. Thanks!

**Comment by huningxin** [comment]
Pushed another commit to expose `CascadeClassifier.empty` and `groupRectangles`. Please take another look @ganwenyao 

For 

> imwrite,imread in test_io.js should be exposed but not
> BackgroundSubtractorMOG2 in test_video.js has error

Please file issue for each one. Thanks! @ganwenyao

**Comment by ganwenyao** [comment]
Looks good

**Comment by huningxin** [comment]
Thanks for review and open new issues.

**Pull request merged**